draft-ietf-jose-fully-specified-algorithms.xml

```diff
@@ -1025,10 +1025,16 @@
 	for ECDSA and EdDSA in <xref target="RFC9053"/> apply.
       </t>
       <t>
-	The security considerations for preventing cross-mode attacks
+	The security considerations for preventing cross-protocol attacks
 	described in <xref target="RFC9459"/> apply.
       </t>
 
+			<t>
+  The list of supported algorithms could be used to produce a fingerprint.
+	Varying the supported algorithms could impact attempts to use fingerprinting.
+	For example, in COSE, an endpoint could alternate between advertising ES256 and ESP256, while producing signatures from the same key.
+      </t>
```

Comment:
```suggestion
      </t>
      <t>
Note that there may be attacks possible with polymorphic algorithms
when the attacker is the party producing a signature.
In this case, the attacker using a polymorphic signature algorithm could vary
the actual algorithm used by changing some of the algorithm parameters
that are not fully-specified by the polymorphic algorithm identifier.
While the signature with different parameters would still be valid,
the recipient might not support the necessary algorithm parameters to validate it.
These kinds of attacks are not possible with fully-specified algorithms.
      </t>
```

Reply:
I moved your suggestion from here: https://github.com/ietf-wg-jose/draft-ietf-jose-fully-specified-algorithms/pull/30/files#r2023852204

So we can keep both, and unpack them with more clarity